### PR TITLE
Quiz report (Teacher)

### PR DESCRIPTION
**Pull request by iAlanx3**

### CODE REVIEW

File: apps/web/src/app/dashboard/teacher/quiz-reports/_components/Cards.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+};
+
+/*
+    <Button variant="outline" size="icon" 
+              onClick={() => {
+                window.open(`/dashboard/teacher/quiz-reports/${item.id}`,);
+                }}>
+                <SizeIcon/>
+              </Button>
+*/
```
Comment (Zenteh-NTU) [inline apps/web/src/app/dashboard/teacher/quiz-reports/_components/Cards.tsx]: Remove this

File: apps/web/src/app/dashboard/teacher/quiz-reports/[slug]/_components/StudentDropDownBar.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        ))}
+      </DropdownMenuContent>
+    </DropdownMenu>
+  );
+};
+/**
+ * {quizreport_topics.teacher_quizreports_topics.map((item) => (
+                            <DropdownMenuItem onSelect={() => setTopic(item.topic)}>{item.topic}</DropdownMenuItem>
+                        ))}
+ */
```
Comment (Zenteh-NTU) [inline apps/web/src/app/dashboard/teacher/quiz-reports/[slug]/_components/StudentDropDownBar.tsx]: Remove this

**Review by Zenteh-NTU — CHANGES_REQUESTED** [review]

File: apps/web/src/app/dashboard/teacher/quiz-reports/_components/Cards.tsx (hunk 1 above)
Comment (iAlanx3) [reply apps/web/src/app/dashboard/teacher/quiz-reports/_components/Cards.tsx]: Removed

File: apps/web/src/app/dashboard/teacher/quiz-reports/[slug]/_components/StudentDropDownBar.tsx (hunk 2 above)
Comment (iAlanx3) [reply apps/web/src/app/dashboard/teacher/quiz-reports/[slug]/_components/StudentDropDownBar.tsx]: Removed

**Review by Zenteh-NTU — APPROVED** [review]

**Pull request merged**